### DISCUSSION TITLE
No __pypackage__ is generated?

### DISCUSSION THREAD

**Discussion by shaolun-huang** [pdm-project/pdm#2052] [Q&A] [answered]
HI,

I just installed pdm from Homebrew. Basically, every instruction says by typing `pdm init`, I will get `__pypackage__` in the current directory. However, mine does not create `__pypackage__`. I am using `pyenv `and `mambaforge` on a M1 Mac at this point, but I think these two should to affect the creation of `__pypackage__`. How can I fix the issue?

**Comment by pawamoy** [comment]
Did you run `pdm install`?

**Reply by shaolun-huang** [reply]
After `pdm add`, I did try `pdm install`. No `__pypackage__` is shown. By the way, I added `mypy`, `pytest `and `pytest-cov`. Hope this is more specific.

**Reply by shaolun-huang** [reply]
If you need any further information, just let me know.

**Reply by pawamoy** [reply]
Do you have a virtualenv activated? What does `pdm use ...` say? For example for me it says this:

```console
% pdm use -f python3.9
Using the last selection, add '-i' to ignore it.
Using Python interpreter: /home/pawamoy/.basher-packages/pyenv/pyenv/shims/python3.9 (3.9)
Using __pypackages__ because non-venv Python is used.
Updating executable scripts...
```

**Reply by shaolun-huang** [reply]
Sorry, I just realized that `__pypackages__` will be used if I choose not to use virtual environment instead. If my understanding is correct, the packages will be installed in the `.venv` directory if the virtual environment is activated from `pdm init`. Otherwise, `__pypackages__` is used.

**Reply by pawamoy** [reply, marked as answer]
Correct :slightly_smiling_face:
